tools/bin/commands/minishift

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+DEFAULT_OPENSHIFT_VERSION="v3.7.1"
+DEFAULT_CPUS="2"
+DEFAULT_DISK_SIZE="20GB"
+DEFAULT_RAM="4912"
+
+set -x
```

Comment:
hey @paoloantinori , is this really necessary to switch on always ? you always can use --verbose ....

Reply:
Despite I actually like it's log4j style of logging in this case it's an honest leftover.
It wasn't supposed to get into.

I'll fix it. any comment instead to the fact that I have removed the ConfigChange trigger ? do I have to restore that too?